reference/api/sortable_attributes.md

```diff
@@ -0,0 +1,98 @@
+# Sortable attributes
+
+_Child route of the [settings route](/reference/api/settings.md)._
+
+Attributes that can be used together with the [`sort` search parameter](/reference/features/search_parameters.md#sort). [You can learn more about sorting in our dedicated guide.](/reference/features/sorting.md)
+
+Sortable attributes can also be updated through the [global settings route](/reference/api/settings.md#update-settings).
+
+## Get sortable attributes
+
+<RouteHighlighter method="GET" route="/indexes/:index_uid/settings/sortable-attributes" />
+
+Get an index's [`sortableAttributes`](/reference/features/sorting.md).
```

Comment:
```suggestion
Get the index's [`sortableAttributes`](/reference/features/sorting.md).
```

Reply:
Since `sortableAttributes` is plural, but I'm not sure!

Reply:
Both could work because the article (`a`/`the`) needs to agree with `index`, not `sortableAttributes`. I also generally tend to think of settings as singular: "get an index's `sortableAttributes` setting" vs "get all sortable attributes belonging to an index."

I have preferred `a/an` in cases like this because an instance can have multiple indexes, each of which can have different settings. What do you think?